kolla/node_custom_config/horizon/custom_local_settings

```diff
@@ -70,3 +70,36 @@ CHAMELEON_PORTAL_SSO_LOGIN_PATH = '/sso/horizon/'
 CHAMELEON_PORTAL_SSO_LOGOUT_PATH = '/logout/'
 SSO_CALLBACK_HOST = '{{ kolla_external_fqdn }}'
 {% endif %}
+
+# A dictionary of settings which can be used to provide the default values for
+# properties found in the Launch Instance modal.
+LAUNCH_INSTANCE_DEFAULTS = {
```

Comment:
Very cool! Is it possible to specify a default flavor, do you know?

Reply:
The only param about flavor is CREATE_INSTANCE_FLAVOR_SORT https://docs.openstack.org/horizon/rocky/configuration/settings.html#create-instance-flavor-sort, which is used to sort the flavors when launching an instance. I tried to set the key to name and hope the first one (we only have one) would be the default selection, but doesn't work.